pysat/__init__.py

```diff
@@ -47,83 +47,53 @@
 logger.addHandler(handler)
 logger.setLevel(logging.WARNING)
 
-# file lock timeout (seconds)
-file_timeout = 10
+# Import and set user and pysat parameters object
+from pysat import _params
 
 # set version
 here = os.path.abspath(os.path.dirname(__file__))
 version_filename = os.path.join(here, 'version.txt')
 
-with Lock(version_filename, 'r', file_timeout) as version_file:
-    __version__ = version_file.read().strip()
-
-# get home directory
+# Get home directory
 home_dir = os.path.expanduser('~')
+
+# Set pysat directory path in home directory
+pysat_dir = os.path.join(home_dir, '.pysat')
+
 # Set directory for test data
 test_data_path = os.path.join(here, 'tests', 'test_data')
-# set pysat directory path in home directory
-pysat_dir = os.path.join(home_dir, '.pysat')
-# make sure a pysat directory exists
-if not os.path.isdir(pysat_dir):
-    # create directory
-    os.mkdir(pysat_dir)
-    print('Created .pysat directory in user home directory to store settings.')
-    # create file with default data directory
-    data_dir = ''
-    if (os.environ.get('TRAVIS') == 'true'):
-        data_dir = '/home/travis/build/pysatData'
-    data_path_file = os.path.join(pysat_dir, 'data_path.txt')
-    with Lock(data_path_file, 'w', file_timeout) as fout:
-        fout.write(data_dir)
 
-        # in case of network files system
-        fout.flush()
-        os.fsync(fout.fileno())
+# Create a .pysat directory or parameters file if one doesn't exist
+if not os.path.isdir(pysat_dir) or \
+        (not os.path.isfile(os.path.join(pysat_dir, 'pysat_settings.json'))):
 
-    print(''.join(("\nHi there!  Pysat will nominally store data in the "
-                   "'pysatData' directory at the user's home directory level. "
-                   "Run pysat.utils.set_data_dir to specify a different "
-                   "top-level directory to store science data.")))
+    # Make a .pysat directory if not already present
+    if not os.path.isdir(pysat_dir):
+        os.mkdir(pysat_dir)
+        os.mkdir(os.path.join(pysat_dir, 'instruments'))
+        os.mkdir(os.path.join(pysat_dir, 'instruments', 'archive'))
+        print('Created .pysat directory in home directory to store settings.')
```

Comment:
Should this go to logging?  Most routines don't tell the user they're doing this.